fetchmail/fetchmail.py

```diff
@@ -45,7 +44,8 @@ def run(connection, cursor):
     for line in cursor.fetchall():
         fetchmailrc = ""
         user_email, protocol, host, port, tls, username, password = line
-        options = "options ssl" if tls else ""
+        options = "options fetchall antispam 501, 504, 550, 553, 554"
```

Comment:
Some more thoughts on fetchall: 

Here's the poll command for my existing setup:
'''
poll mail.solwiz.ch proto imap
   user 'test@solwiz.ch' there with password 'aSecretNeverToBeTold'
   is 'test@solwiz.test' here
   options fetchall antispam 501, 504, 550, 553, 554
   sslproto 'AUTO'
'''

There's an option '''nokeep''', which seems to be the default, which means messages are deleted once fetched. I don't get duplicates with fetchmail, but that's the only process reading the mailbox.